target/linux/armvirt/image/Makefile

```diff
@@ -19,10 +19,23 @@ define Image/Build/Initramfs
 	)
 endef
 
+define Image/Build/gzip
+	gzip -f9n $(BIN_DIR)/$(IMG_PREFIX)-root.$(1)
```

Comment:
`gzip -f9n $(BIN_DIR)/$(IMG_PREFIX)-rootfs-$(1).img` as well please.

Reply:
Let's figure out malta before adjusting this.